modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -138,3 +138,13 @@ If guest users do not see published API versions that have been set to be visibl
 . Open Salesforce setup.
 . Open the sharing rules.
 . Ensure that the sharing rule for the object `CommunityApi` matches your community name exactly, including spacing and letter case.
+
+== Refresh the Cache
+
+If changes made to APIs are not yet visible, you can refresh the cache to see them.
+
+. Log in to your Salesforce organization.
+. In *Setup*, click *Platform Cache*.
+. Open the `ACMCache` partition.
+. If the caches are in use, then the *Clear Cache* button is displayed.
+. Click *Clear Cache*.
```

Comment:
Clear Cache doesn't affect the recent changes? This sounded kind of scary to me; do we need to say that this clears the previous cache and displays the updated cache?

Reply:
don't think it's necessary - this just clears the cache. But bear in mind the cache is just a copy of the data - if it's deleted, it will be populated again automatically as data is accessed.